pkg/datagatherer/local/local.go

```diff
@@ -46,3 +46,12 @@ func (g *DataGatherer) Fetch() (interface{}, error) {
 	}
 	return dataBytes, nil
 }
+
+// ConfigError is the error type for a misconfiguration. e.g. A missing CRD
```

Comment:
`local` is not the right package for this. I suggest moving this to `datagatherer`.